torch/utils/data/datapipes/dataframe/dataframe_wrapper.py

```diff
@@ -1,5 +1,7 @@
-_pandas = None
-_WITH_PANDAS = None
+from typing import Any, Optional
+
+_pandas: Any = None
```

Comment:
```suggestion
_pandas: Optional = None
```

Reply:
So this I *know* will flag a rule somewhere saying that you shouldn't omit type parameters from generics...

Reply:
https://mypy.readthedocs.io/en/stable/config_file.html#confval-disallow_any_generics

Reply:
Oh this triggers for Type too

```
$ mypy --disallow-any-generics a.py
a.py:3: error: Missing type parameters for generic type "Type"  [type-arg]
Found 1 error in 1 file (checked 1 source file)
```

Reply:
Ah, makes sense. Cary on then.